Examples/Python/tests/helpers/__init__.py

```diff
@@ -30,6 +30,13 @@
             "ROOT likely built without/with incompatible PyROOT. Skipping tests that need ROOT"
         )
 
+try:
+    from acts import geomodel
+
+    geomodelEnabled = True
+except ImportError:
+    geomodelEnabled = False
+
```

Comment:
This might not work correctly, as the `geomodel` module is not a *proper* sub module. That means the error is not an `ImportError` but something like a missing attribute error. This is the case for example for the `pythia8` module below.

You could do



```suggestion
try:
    import acts

    geomodelEnabled = hasattr(acts, "geomodel")
except ImportError:
    geomodelEnabled = False

```